### PR TITLE
Add new template options

### PR DESCRIPTION
**Pull request by admmasters**

### CODE REVIEW

File: src/components/styles/Theme/ThemeProvider.tsx [hunk 1]
```diff
@@ -235,7 +235,8 @@ export interface SpinnerTheme {
   };
 }
 
-export interface TypographyTheme {
+export interface TypographyTheme<Font = any> {
+  font?: Font;
```
Comment (admmasters) [inline src/components/styles/Theme/ThemeProvider.tsx:239]: Frameworks such as nextjs have their own font abstractions.

File: src/components/templates/ProductTemplate/ProductTemplate/ProductTemplateV2.tsx [hunk 2]
```diff
@@ -7,12 +7,16 @@ import FlexRow from '../../../layout/FlexRow/FlexRow';
...
 import styled from 'styled-components';
+import { useThemeContext } from '../../../styles/Theme';
+import { Text } from '../../../..';
 
 export interface ProductTemplateV2 {
-  backgroundColor: string;
-  header: {
-    backgroundColor: string;
-    height: string;
+  titleClassName: string;
```
Comment (admmasters) [inline src/components/templates/ProductTemplate/ProductTemplate/ProductTemplateV2.tsx:14]: All of the underlying elements can be targeted by utility classes to override them - supports Tailwind

File: src/components/templates/ProductTemplate/ProductTemplate/ProductTemplateV2.tsx [hunk 3]
```diff
@@ -66,4 +70,22 @@ function ProductTemplate({ header, children, contentWidth = 6 }: ProductTemplate
   );
 }
 
-export { ProductTemplate };
+type PreLabelProps = {
+  title: string;
+  subtitle: string;
+};
+
+const PreLabel = ({ title, subtitle }: PreLabelProps) => {
```
Comment (admmasters) [inline src/components/templates/ProductTemplate/ProductTemplate/ProductTemplateV2.tsx:78]: I'm not sure if we should move this to a molecule, probably should do.

**Review by OlenaKashuba — COMMENTED** [review]
changeset needed

File: src/index.ts [hunk 4]
```diff
@@ -13,7 +13,7 @@ export {
   mobileNavbarHeight,
 } from './constants';
 export { default as GlobalStyles } from './components/styles/GlobalStyles';
-export { ThemeProvider } from './components/styles/Theme';
+export { ThemeProvider, zopaTheme } from './components/styles/Theme';
```
Comment (admmasters) [inline src/index.ts:16]: We now export the `zopaTheme` this lets someone extend it easily and allows a more consistent return type in client applications.

Comment (OlenaKashuba) [reply src/index.ts:16]: amazing, we might need that in rezza

File: src/components/templates/ProductTemplate/ProductTemplate/ProductTemplateV2.tsx (hunk 3 above)
Comment (OlenaKashuba) [reply src/components/templates/ProductTemplate/ProductTemplate/ProductTemplateV2.tsx:78]: whats prelabel visually? text that goes on top of progress?

**Review by zopa-charlotte-walker — APPROVED** [review]

**Review by zopa-charlotte-walker — APPROVED** [review]

**Pull request merged**